CMakeLists.txt

```diff
@@ -63,8 +63,10 @@ set(DOXYGEN_USE_TEMPLATE_CSS YES)
 
 # Use BoostConfig module distributed by boost library instead of using FindBoost module distributed
 # by CMake
-if(POLICY CMP0167)
-  cmake_policy(SET CMP0167 NEW)
+if(NOT WIN32 OR NOT ${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
```

Comment:
```suggestion
if(WIN32 AND ${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
```

Reply:
I'm not sure the suggestion works. We want the combination Win+Clang to not set CMP0167. Therefore, the proposed logic allows all other setups (combinations of platform + compiler) to pass the first if and check if CMP0167 is available.

Reply:
You're totally right. 'Je n'ai pas les yeux en face des trous'